src/doc/building_visit/Building_Directly_With_CMake.rst

```diff
@@ -0,0 +1,20 @@
+.. _Building Directly with CMake:
+
+Building Directly with CMake
+----------------------------
+If the third party libraries have already been installed using ``build_visit``, 
+VisIt_ can be built without the use of the ``build_visit`` script, with these 
+steps.
+
+.. code:: bash
+
+  git clone git@github.com:visit-dav/visit.git
+  mkdir visit/build
+  cd visit/build
+
+``build_visit`` should have created a cmake file specific to your machine.
+CMake needs to be simply pointed at it. Examples can be found in the 
+``src/config-site`` directory.
```

Comment:
Can you make the ``src/config-site`` text here a clickable link to our repository's config-site dir?

Reply:
How can I do that? Or alternatively, where can I find a good example of it?

Reply:
Hmmm... 🤔 maybe scan through some pages of the hosted docs like [this one](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/ContributingToDocumentation/index.html#about-line-length) looking for cases where you see links out of the docs to somewhere. Then scroll to top of those pages in browser and hit the `Edit on GitHub` button in upper right. That'll get you to that file. Then, finally, view the **Raw** file (GitHub tries to interpret `.rst` files so viewing **Raw** is helpful in allowing you to see the actually ascii source).

All that said, there are a few bullets describing how links work in our doc's [Contributing to documentation](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/ContributingToDocumentation/index.html#about-line-length) section.

If after trying any of that, it still isn't obvious, ping again here.